docs/server_api/index.rst

```diff
@@ -1,5 +1,17 @@
-Bodhi REST API
-==============
+=================
+Bodhi Server APIs
+=================
+
+fedmsg API
+==========
+
+You can read about the fedmsgs that Bodhi sends here:
```

Comment:
You could write this something like:
```
The fedmsgs bodhi sends `are documented in the fedora fedmsg project<https://fedora-fedmsg.readthedocs.io/en/latest/topics.html#bodhi>`_ 
```

Just a suggestion. You do what you think is right.